ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__hcp-cnf-4.17.yaml

```diff
@@ -0,0 +1,118 @@
+base_images:
+  ansible:
+    name: "4.17"
+    namespace: ocp
+    tag: ansible
+  assisted-image-service:
+    name: ocm-2.12
+    namespace: edge-infrastructure
+    tag: assisted-image-service
+  assisted-installer:
+    name: ocm-2.12
+    namespace: edge-infrastructure
+    tag: assisted-installer
+  assisted-installer-agent:
+    name: ocm-2.12
+    namespace: edge-infrastructure
+    tag: assisted-installer-agent
+  assisted-installer-controller:
+    name: ocm-2.12
+    namespace: edge-infrastructure
+    tag: assisted-installer-controller
+  assisted-service:
+    name: ocm-2.12
+    namespace: edge-infrastructure
+    tag: assisted-service
+  assisted-test-infra:
+    name: ocm-2.12
+    namespace: edge-infrastructure
+    tag: assisted-test-infra
+  aws-ebs-csi-driver-operator-test:
+    name: "4.17"
+    namespace: ocp
+    tag: aws-ebs-csi-driver-operator-test
+  azure-disk-csi-driver-operator-test:
+    name: "4.17"
+    namespace: ocp
+    tag: azure-disk-csi-driver-operator-test
+  base:
+    name: "4.17"
+    namespace: ocp
+    tag: base
+  console-tests:
+    name: "4.17"
+    namespace: ocp
+    tag: console-tests
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  driver-toolkit-tests:
+    name: "4.17"
+    namespace: ocp
+    tag: driver-toolkit-tests
+  gcp-pd-csi-driver-operator-test:
+    name: "4.17"
+    namespace: ocp
+    tag: gcp-pd-csi-driver-operator-test
+  ibm-vpc-block-csi-driver-operator-test:
+    name: "4.17"
+    namespace: ocp
+    tag: ibm-vpc-block-csi-driver-operator-test
+  libvirt-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: libvirt-installer
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+  vsphere-csi-driver-operator-test:
+    name: "4.17"
+    namespace: ocp
+    tag: vsphere-csi-driver-operator-test
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.17"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-telcov10n-functional-hcp-cnf-stable
+  cluster: build05
+  cron: 00 18 * * 5
+  steps:
+    env:
+      T5CI_JOB_TYPE: hcp-cnftests
+      T5CI_VERSION: "4.17"
+    observers:
+      enable:
+      - observers-resource-watch
+    workflow: telcov10n-functional-hcp-cnf
+- as: e2e-telcov10n-functional-hcp-cnf-nightly
```

Comment:
Thats a name, please inlude it on .config.prowgen.

Reply:
np will do.